### ISSUE TITLE
Peerjs server deployed on heroku is not working

### ISSUE THREAD

**Issue by Chhinna** [peers/peerjs-server#207]
I used the deploy on Heroku button from this [link](https://elements.heroku.com/buttons/peers/peerjs-server) to deploy a peerjs server. To connect to this server I used 

`const peer = new Peer(host: 'focusmonk-beta.herokuapp.com', port:9000, path:'/')`

But this is not working, am I missing something ?

**Comment by vishalveerareddy** [comment]
That's because the default password is wrong. Trying to find a fix

**Comment by RamyaAshika** [comment]
Any update on this, its saying connected but not receiving connection on two peers ? @vishalveerareddy  @Chhinna

**Comment by ElmouradiAmine** [comment]
you need to use the port 443 I had the same issue. But I encountered another one :(. Due to security issues I cannot send my getUserMedia stream over the peer js connection.

**Comment by siddharthmagadum16** [comment]
I'm facing a similar issue.  Is there any fix for this or not? 
The request for `https://<server-name>.herokuapp.com:9000/peer-server/peerjs/id?ts=16312062773910.47823913209901503`  is failing due to CORS request did not succeed error.
How to fix this?

**Comment by blyzniuk** [comment]
> I'm facing a similar issue. Is there any fix for this or not?
> The request for `https://<server-name>.herokuapp.com:9000/peer-server/peerjs/id?ts=16312062773910.47823913209901503` is failing due to CORS request did not succeed error.
> How to fix this?

As Amine said above, you need to set 433 port in your url when you use https connection, or 80 port for http. This is default TCP ports.

**Comment by siddharthmagadum16** [comment]
> > I'm facing a similar issue. Is there any fix for this or not?
> > The request for `https://<server-name>.herokuapp.com:9000/peer-server/peerjs/id?ts=16312062773910.47823913209901503` is failing due to CORS request did not succeed error.
> > How to fix this?
> 
> As Amine said above, you need to set 433 port in your url when you use https connection, or 80 port for http. This is default TCP ports.

Yeah, we need to use port 443, otherwise, it won't work. I ended up using the 0.peerjs.com peer cloud server instead of creating my own peer server.
